DQMOffline/L1Trigger/interface/L1TDiffHarvesting.h

```diff
@@ -0,0 +1,63 @@
+#ifndef DQMOFFLINE_L1TRIGGER_L1TDIFFHARVESTING_H
+#define DQMOFFLINE_L1TRIGGER_L1TDIFFHARVESTING_H
+
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/LuminosityBlock.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+
+#include "DQMServices/Core/interface/DQMEDHarvester.h"
+#include "DQMServices/Core/interface/DQMStore.h"
+#include "DQMServices/Core/interface/MonitorElement.h"
+
+#include <string>
+#include <vector>
+
+namespace dqmoffline {
+namespace l1t {
+
+class L1TDiffHarvesting: public DQMEDHarvester {
+
+public:
+  L1TDiffHarvesting(const edm::ParameterSet& ps);
+  virtual ~L1TDiffHarvesting();
+
+protected:
+
+  virtual void dqmEndJob(DQMStore::IBooker &ibooker, DQMStore::IGetter &igetter) override;
+  virtual void dqmEndLuminosityBlock(DQMStore::IGetter &igetter, edm::LuminosityBlock const& lumiBlock,
+      edm::EventSetup const& c);
+
+private:
+  class L1TDiffPlotHandler {
+  public:
+    L1TDiffPlotHandler(const edm::ParameterSet & ps, std::string plotName);
+    L1TDiffPlotHandler(const L1TDiffPlotHandler &handler); // needed for vector collection
+
+    void computeDiff(DQMStore::IBooker &ibooker, DQMStore::IGetter &igetter);
+
+    std::string dir1_;
+    std::string dir2_;
+    std::string outputDir_;
+    std::string plotName_;
+
+    MonitorElement* h1_;
+    MonitorElement* h2_;
+    MonitorElement* h_diff_;
+    MonitorElement::Kind histType1_, histType2_;
+
+    void loadHistograms(DQMStore::IGetter &igetter);
+    bool isValid() const;
+    void bookDiff(DQMStore::IBooker &ibooker);
+
+  };
+
+  typedef std::vector<L1TDiffPlotHandler> L1TDiffPlotHandlers;
+
+  std::vector<edm::ParameterSet> plotCfgs_;
```

Comment:
Looks like this is only used in the constructor - no need to keep it in the class definition